crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -80,6 +80,12 @@ def cancel
     render json: { error: e.message }, status: :unprocessable_entity
   end
 
+  api :GET, "/api/upgrade/repocheck", "Check for missing node repositories"
```

Comment:
So, this was raised before by @vuntz , shouldn't we rather call it differently than repocheck for admin server repositories?